pkg/kn/commands/source/human_readable_flags.go

```diff
@@ -29,7 +29,8 @@ var sourceTypeDescription = map[string]string{
 	"ApiServerSource": "Kubernetes API Server events source",
 	"ContainerSource": "Container events source",
 	"CronJobSource":   "CronJob events source",
-	"SinkBinding":     "Binding Pattern for ContainerSource",
+	"SinkBinding":     "Binding pattern for linking Sink to PodSpecable",
+	"PingSource":      "Scheduled ping events source",
```

Comment:
A bit of a nit-pick, but I think we should be a more descriptive here instead of repeating the name:

```suggestion
	"PingSource":      "Sending periodically ping events to a sink",
```

Reply:
And then for the other descriptions

```
ApiServerSource: Watching for Kubernetes API events and then them to a sink
ContainerSource: Connecting a custom container image with a sink
CronJobSource: Sending periodically constant data to a sink
```

(not sure about whether to use the gerund ("Sending") or an imperative ("Send"). Not really good in english grammar ;-)

Reply:
@rhuss : I've updated as below:
```
Watch and send Kubernetes API events to a sink
Connect a custom container image to a sink
Send periodically ping events to a sink
Binding for connecting a PodSpecable to a sink
```